### PR TITLE
modificari tictactoe

### PR DESCRIPTION
**Pull request by Slidem**
Hey, made some modifications

### CODE REVIEW

File: src/utils/Joc.java [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package utils;
+
+public interface Joc {
```
Comment (Slidem) [inline src/utils/Joc.java:3]: Interface not necessary, only left as an example

**Comment by razvanmacovei15** [comment]
hei mate, thank you for the help

File: src/utils/Joc.java (hunk 1 above)
Comment (Slidem) [reply src/utils/Joc.java:3]: Ok, got it

**Review by razvanmacovei15 — APPROVED** [review]
all is good

**Pull request merged**